### ISSUE TITLE
Audio crashes on boot 

### ISSUE THREAD

**Issue by unicornsandcandies** [ublue-os/bazzite#1181]
### Describe the bug

When starting bazzite, audio crashes and keeps stuttering until reboot.

Logs on a bad start:
[f294f143.txt](https://github.com/ublue-os/bazzite/files/15448541/f294f143.txt)

Logs immediately after reboot and launching a game (good start):
[68a5d69e.txt](https://github.com/ublue-os/bazzite/files/15448546/68a5d69e.txt)

### What did you expect to happen?

Audio outputs without crashes.

### Output of `rpm-ostree status`

```shell
Included in logs
```

### Hardware

Galileo
Steam Deck OLED

**Comment by KyleGospo** [comment]
This was a known bug and should be fixed in the latest stable release

**Comment by unicornsandcandies** [comment]
> This was a known bug and should be fixed in the latest stable release

It's happening to me on the latest stable release (40.20240524.0)

**Comment by noelmiller** [comment]
Is the audio crashing completely or is the audio stuttering and distorted? I haven't gotten that behavior to replicate again, but I was having it before the most recent update.

**Comment by unicornsandcandies** [comment]
> Is the audio crashing completely or is the audio stuttering and distorted? I haven't gotten that behavior to replicate again, but I was having it before the most recent update.

It's stuttery and distorted. It may not be easy to replicate quickly, because I observed it only on some boots. Sometimes it works okay but only until I run a game (tested on Like a dragon and Hogwarts Legacy) and then it crashes.

**Comment by noelmiller** [comment]
> > Is the audio crashing completely or is the audio stuttering and distorted? I haven't gotten that behavior to replicate again, but I was having it before the most recent update.
> 
> It's stuttery and distorted. It may not be easy to replicate quickly, because I observed it only on some boots. Sometimes it works okay but only until I run a game (tested on Like a dragon and Hogwarts Legacy) and then it crashes (I have a music in my deck menu with decky plugin and it works alright, issues start when starting a game), most of the time it only requires a bad boot and the whole os, including system sounds are stuttery.
> 
> I've managed to catch it. Hopefully the logs will help with identifying the problem.
> 
> I've also checked to see if there's any update I may have missed and it seems that I'm on the latest version.

Yeah, stuttery and distorted is what I was getting too. I haven't had it happen to me since updating to the latest `:stable` image. It's more intermittent now than it was before if it's still happening.
